src/main/resources/templates/start.html

```diff
@@ -5,6 +5,6 @@
     <title>Caching In</title>
 </head>
 <body>
-Geo-caching all over the world!
+<h1>Geo-caching all over the world!</h1>
```

Comment:
small error but just added <h1> tags so it would help the text out more.

Reply:
This was covered in a previous review.  When that occurs, first in wins.